source/features/repo-icon-buttons.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import PlusIcon from 'octicon/plus.svg';
+import SearchIcon from 'octicon/search.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {groupButtons} from '../github-helpers/group-buttons';
+
+function init(): void {
+	const searchButton = select('.file-navigation a.btn.d-md-block')!;
+	searchButton.classList.remove('mr-2');
+	searchButton.childNodes[0].replaceWith(<SearchIcon/>);
+
+	const addButton = select('.file-navigation .d-md-flex.flex-items-center')!;
+	addButton.classList.replace('d-md-flex', 'd-md-block');
+	addButton.classList.remove('ml-2');
+	addButton.childNodes[0].replaceWith(<PlusIcon/>);
```

Comment:
```suggestion
	empty(addButton);
	addButton.append(<PlusIcon/>);
```

It should also drop the arrow. `empty` is part of dom-utils

Reply:
If you are referring to emptyElement that was removed https://github.com/sindresorhus/refined-github/commit/904b1a00398b73d35eee741ac4820911a59b7200

Reply:
As @yakov116 pointed out I end up removing the caret manually